### DISCUSSION TITLE
[Design] Breaking The Knockdown Meta

### DISCUSSION THREAD

**Discussion by PopeDaveThe3th** [ParadiseSS13/Paradise#25974] [Documents]
### The Problem

Paradise melee combat is built around, and dominated by, knockdowns. Weapons which are capable of knocking a target down are, generally, going to automatically win in the event they are faced with a weapon that cannot. The most obvious culprit here is the stun baton, which despite being the standard weapon of security players, can instantly secure a victory in melee with a single hit thanks to its knockdown.

This leads to the next issue, that of the "adrenals tax". Essentially, for any traitor (or antag generally) that wants to combat security directly, adrenals are pretty much a requirement. Adrenals allow you to instantly leave the knockdown state and run away from your would-be captors in an instant. This, of course, presents its own host of problems. Firstly, it encourages security to use lethal force more often, as antagonists are virtually guaranteed to be using some form of anti-stun. It also feeds into the "speed meta", which itself is an entirely separate can of worms to be addressed at a later date.

This then leads to the issue that now _everything_ (mild hyperbole) is now balanced around the power of the stun baton. Weapons like the chainsaw, ebow, armblade, and contractor baton are quite literally infinite sources of knockdown. Antagonist melee weapons in particular _need_ to be incredibly powerful in order to have a chance when faced with the sheer power of the mallcop stunstick. 

**TLDR:** Excessively common knockdown-capable weapons are strangling melee combat by forcing a certain playstyle to survive against them, and have taken over balance concerns regarding new or updated weaponry and armor.

Additionally, it really just isn't fun to sit on the floor helpless after one lucky baton hit.

**Now comes the hard part. How do we stop this?**

An excellent question, dear reader.

### The Solution

I'll be honest, I created this document in part to try and _find_ solutions to this conundrum. 

Obviously, we could just remove all knockdowns altogether. It's certainly an approach I favor for certain items.

We could also add additional ways to mitigate or otherwise avoid stuns, though I feel this avoids the root issue of stuns oppressing melee combat.

Maybe we could make knockdowns less punishing, but I feel this probably isn't possible while keeping combat somewhat reasonable.

Or perhaps I'm just an idiot who doesn't know what he's talking about. Any and all feedback regarding this issue is greatly appreciated.

**Comment by CRUNCH-Borg** [comment]
I always found it extremely silly that you drop held items on knockdown like everything is covered in lube. Why do you need an implant just to have sufficient grip strength to hold onto something when someone shoves you into a flat wall? Why does every spessman have carpel tunnel syndrome?

Removing the instant disarm of knockdowns would make them far far less punishing and also just make more sense.

**Comment by FaronCD** [comment]
Writing what marmite wrote since he did a pretty good job of balance. “Make stunbatons four hits and disablers six”

**Reply by FaronCD** [reply]
Balancing*
